ctl_main.c

```diff
@@ -1164,6 +1170,17 @@ static int cmd_setportdonttxmt(int argc, char *const *argv)
     return set_port_cfg(dont_txmt, getyesno(argv[3], "yes", "no"));
 }
 
+static int cmd_setportbpdufilter(int argc, char *const *argv)
+{
+    int br_index = get_index(argv[1], "bridge");
+    if (0 > br_index)
+        return br_index;
+    int port_index = get_index(argv[2], "port");
```

Comment:
maybe declare `int port_index;` at the beginning of the function;
this is for cases where some people may use some old compilers, or compiler options that don't enable C99

Reply:
Maybe it should be declared that way in the rest of the codebase then?